assets/sendmail_template.txt

```diff
@@ -59,4 +59,30 @@ Content-Disposition: inline; filename="NGI_logo.png"
   flatten().
   join( '\n' ) %>
 
---ngimimeboundary--
+<%
+if (mqcFile){
+def mqcFileObj = new File("$mqcFile")
+if (mqcFileObj.length() < 25000000){
+out << """
+--nfcoremimeboundary
+Content-Type: application; name=\"multiqc_report\"
```

Comment:
The correct MIME content-type should be `text/html` according to https://www.iana.org/assignments/media-types/media-types.xhtml#text